.github/workflows/synchronizing.yaml

```diff
@@ -0,0 +1,137 @@
+                mv aws-integrations/lambda-integrations/$integration/template.yaml template-directory/${integration}-template.yaml
+                echo $integration >> changed-integration.txt
+              fi
+            fi
+          done
+          if [ -z "$(ls -A "./template-directory")" ]; then
+            echo "[INFO] No file in the integrations list where changed."
+            exit 1
+          fi
+
+      - name: Upload template-directory
+        uses: actions/upload-artifact@v2
+        with:
+          name: template-directory
+          path: ./template-directory/
+
+      - name: Upload changed-integration
+        uses: actions/upload-artifact@v2
+        with:
+          name: changed-integration
+          path: ./changed-integration.txt
+  
+      - name: Upload replace_template script
+        uses: actions/upload-artifact@v2
+        with:
+          name: replace_template
+          path: ./replace_template.sh
+
+  create_pr:
+    runs-on: ubuntu-latest
+    needs: upload_files
+    steps:
+
+      - name: Checkout destination repository
+        uses: actions/checkout@v2
+        with:
+          repository: coralogix/integration-definitions
+          token: ${{ secrets.GH_TOKEN }}
+
+      - name: Create template-directory
+        run: mkdir ./template-directory
+
+      - name: Download template-directory
+        uses: actions/download-artifact@v3
+        with:
+          name: template-directory
+          path: ./template-directory
+    
+      - name: Download replace_template file
+        uses: actions/download-artifact@v3
+        with:
+          name: replace_template
+          path: ./
+
+      - name: Download changed-integration file
+        uses: actions/download-artifact@v3
+        with:
+          name: changed-integration
+          path: ./
+
+      - name: apply changes to the files
+        run: |
+          git pull origin main
+          git checkout -b sync-branch
+          sync_branch_exists=$(git ls-remote --heads origin "sync-branch" | wc -l)
+          if [[ $sync_branch_exists -ne 0 ]]; then
+            git pull origin sync-branch --rebase
+          fi
+          chmod +x replace_template.sh
+          while read line; do  
+            ./replace_template.sh template-directory/${line}-template.yaml
+            template_version=$(cat template-directory/${line}-template.yaml | grep "SemanticVersion" | grep -oE '[^ ]+$')
+            mkdir -p integrations/${line}/v${template_version}
+            mv template-directory/${line}-template.yaml integrations/${line}/v${template_version}/template.yaml
+            
+            if [ -f integrations/${line}/manifest.yaml ]; then
+              echo "  - revision: ${template_version}
+              template: !CloudFormationTemplate v${template_version}/template.yaml
+              field_definitions: v${template_version}/fields.yaml" >> integrations/${line}/manifest.yaml
+            fi
```

Comment:
IIUC, if the file does not exist we will add it manually, for the first time. I'm fine with that 👍

Reply:
Yes the manifest and the fields files you will need to add manually for new integration